sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -174,10 +174,10 @@ namespace Azure { namespace Core { namespace Http {
      *
      * @param httpMethod HTTP method.
      * @param url URL.
-     * @param downloadViaStream A boolean value indicating whether download should happen via
-     * stream.
+     * @param noBufferDownload A boolean value indicating whether download should not use a buffer
+     * for the response but provide a body stream instead.
```

Comment:
Reading the negation makes it hard to understand what we are saying.
```suggestion
     * @param noBufferDownload A boolean value indicating whether download should use a buffer
     * for the response or provide a body stream instead.
```